CRM/Core/Form/RecurringEntity.php

```diff
@@ -400,7 +400,7 @@ public static function postProcess($params = [], $type, $linkedEntities = []) {
         //Delete relations if any from recurring entity tables before inserting new relations for this entity id
         if ($params['entity_id']) {
           //If entity has any pre delete function, consider that first
-          if (CRM_Utils_Array::value('pre_delete_func', CRM_Core_BAO_RecurringEntity::$_recurringEntityHelper[$params['entity_table']]) &&
+          if (!empty(CRM_Core_BAO_RecurringEntity::$_recurringEntityHelper[$params['entity_table']]['pre_delete_func']) &&
```

Comment:
There's another instance in the same `if` statement that could also be converted.

Reply:
Fixed